mix.exs

```diff
@@ -27,7 +27,14 @@ defmodule Plenario.Mixfile do
   def application do
     [
       mod: {Plenario.Application, []},
-      extra_applications: [:logger, :runtime_tools, :bamboo, :bamboo_smtp, :sentry]
```

Comment:
Not really related to the functionality of this PR, but we really need to standardize our code style. I'll look into Elixir/Phoenix linting

Reply:
There's `mix fmt`, it's the right way (tm) to format our code. We could even have it reject builds that don't have properly formatted code 😸. I think we just haven't done it yet because it would result in a gorillion lines changed.

Reply:
I'll champion implementing it as part of the move to the umbrella

Reply:
This needs to be discussed more, but I really *hate* some of the things that `mix fmt` does that cannot be undone with the config.